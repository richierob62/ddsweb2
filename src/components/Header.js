import React from 'react'
import styled from 'styled-components'

const StyledH1 = styled.h1`
  background-color: #8e44ad;
  color: rgba(255, 255, 255, 0.7);
  text-align: center;
  padding: 10px 5px;
`

const Header = () => <StyledH1>DDSWeb Directory Data Manager</StyledH1>

export default Header
