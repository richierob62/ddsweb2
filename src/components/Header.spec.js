import React from 'react';
import Header from './Header';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

it('renders without crashing', () => {
  const el = shallow(<Header />);
  expect(shallowToJson(el)).toMatchSnapshot();
});

