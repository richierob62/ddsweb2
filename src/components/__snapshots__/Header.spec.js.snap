exports[`test renders without crashing 1`] = `
<div
  className="Header">
  <h1>
    This is the header
  </h1>
</div>
`;
